patches/@nrk+core-datepicker+3.0.8.patch

```diff
@@ -53,7 +53,30 @@ index 00848d7..5096ed7 100644
          var _button = closest(event.target, 'button[value]');
  
          var _table = closest(event.target, 'table');
-@@ -580,18 +613,34 @@ function table(self, table) {
+ 
+-        if (_button) this.date = _button.value;
++        if (_button) {
++          var date = MASK[_button.getAttribute('data-type')].replace('*', _button.value);
++          var match = _button.value.match(new RegExp(Object.keys(FILL).filter(key => key !== "null").join("\|")));
```

Comment:
Please remind me, blir denne koden transpilert? For filter og arrow functions funker ikke i IE.

Reply:
Jepp, alt blir transpilert 😊